src/v2/guide/components-slots.md

```diff
@@ -58,27 +58,28 @@ If `<navigation-link>`'s template did **not** contain a `<slot>` element, any co
 When you want to use data inside a slot, such as in:
 
 ``` html
+<!-- parent component -->
 <navigation-link url="/profile">
   Logged in as {{ user.name }}
 </navigation-link>
 ```
 
-That slot has access to the same instance properties (i.e. the same "scope") as the rest of the template. The slot does **not** have access to `<navigation-link>`'s scope. For example, trying to access `url` would not work:
+That slot has access to the same instance properties (i.e. the parent "scope") as the rest of the template. The slot does **not** have access to `<navigation-link>`'s scope (i.e. the child "scope"). For example, trying to access `url` would not work:
 
 ``` html
 <navigation-link url="/profile">
   Clicking here will send you to: {{ url }}
   <!--
   The `url` will be undefined, because this content is passed
-  _to_ <navigation-link>, rather than defined _inside_ the
-  <navigation-link> component.
+  _to_ the child component <navigation-link> after `url` is interpolated in parent component,
```

Comment:
**concern**: from my point of view, the term `interpolated` here add to the cognitive load and is not really descriptive, so I would prefer to leave this part as it was before